tests/test_integration_tests.py

```diff
@@ -274,9 +275,9 @@ def test_sustainability_summary_query(self, connection):
         assert beryllium_summary.mass_before_processing.value == pytest.approx(0.027)
         assert beryllium_summary.material_reference.record_guid is not None
         assert beryllium_summary.climate_change.value == pytest.approx(15.52, DEFAULT_TOLERANCE)
-        assert beryllium_summary.climate_change_percentage == pytest.approx(54.32, DEFAULT_TOLERANCE)
+        assert beryllium_summary.climate_change_percentage == pytest.approx(49.25, DEFAULT_TOLERANCE)
```

Comment:
A lot of these values changed, I assume because of changes in the database. I will double check though, don't merge until this thread is resolved.

Reply:
I haven't gone through all the calculations manually, but I have checked the databases and there are differences in eco indicators between versions, so I think this is fine.